submissions/fabunmiTolulope/HTML/Productpage.html

```diff
@@ -4,54 +4,86 @@
+<section class="cavendish">
+	<img src="../html/images/cavendish.jpg";>
+		<div class="cost">
+			<h3> ORGANIC CAVENDISH BANANA</h3>
+			<p>Brand: Organic Cavendish</p>
+			<label for="quantity">Quantity:</label>
+			<input type="number" name="quantity" id="quantity" value="1">
+			<p>Price: &#x20A6;10,000</p>
+			<p>Shipping Cost: &#x20A6;1,500</p>
+			<p>Total Cost: &#x20A6;11,500</p>
+			<button type="submit" id="btn"><a href="#">BUY NOW</a></button>
+		</div>
 </section>
-<section>
-<p><h3> Product Description </h3>
-	From a cultivars that dominate the international trade in bananas, this <strong>Organic giant Cavendish Banana (<em>Musa sapenticum</em>)</strong> sets the standards in terms of taste and nutrient yield. <br>They are rich in Vitamin C, B6, minerals and dietary fibres. They are a rich energy source with 25%  carbohydrate. <br> It could be used for Banana Starch, flour and chips.<br> They can be eaten raw, used as smoothie and added to your homemade yummy yoghurt and Icecream.
-	<h3>Similar breeds of banana available</h3>
-	<ol>
-		<li type="i">Plantain (Cooking banana) </li>
-		<li type="i"> Grand Nain banana</li>
-		<li type="i"> Gros michel banana</li>
-		<li type="i"> Lady finger banana</li>
-		<li type="i"> <a href="#">  More items </a></li>
-	</ol> 
-</p><hr>
+<div class="description">
+	<h3>DESCRIPTION </h3>
+	<p>From a cultivars that dominate the international trade in bananas, this <strong>Organic giant Cavendish Banana (<em>Musa sapenticum</em>)</strong> sets the standards in terms of taste and nutrient yield. <br>They are rich in Vitamin C, B6, minerals and dietary fibres. They are a rich energy source with 25%  carbohydrate. <br> It could be used for Banana Starch, flour and chips.<br> They can be eaten raw, used as smoothie and added to your homemade yummy yoghurt and Icecream. </p>
+</div>
+<section class="similar">
+	<h3>SIMILAR BREEDS OF BANANA</h3>
+		<ul>
+			<li> 
+				<figure>
+				<img src="../html/images/plantain.jpg">
+				<figcaption>Plantain (Cooking banana). <br> Price: &#x20A6;10,000</figcaption>
+				</figure>
+			</li>
+			<li>
+				<figure>
+				<img src="../html/images/gros-michel.jpg">
+				<figcaption> Gros michel banana <br> Price: &#x20A6;10,500 </figcaption>
+				</figure>
+			</li>
+			<li>
+				<figure>
+				<img src="../html/images/lady-finger.jpg">
+				<figcaption> Lady finger banana <br> Price: &#x20A6;8,500 </figcaption>	
+				</figure>
+			</li>
+		</ul> 
 </section>
-<nav>
-	<ul>
-		<li><a href="#"> Add to cart</a></li>
-		<li><a href="#">View more products</a> </li>
-	</ul>
-</nav>
 <footer>
-	<p> For more enquiries: <em>contact us on +2347063740223 or send a mail to us at Greenprint@gmail.com</em>
-		<ul>
-			<li><a href="https://wwww.Instagram.com">Instagram</a></li>
-			<li><a href="https://wwww.facebook.com">facebook</a></li>
-		</ul>
-	</p>
-	<p><big>&copy; 2020 Greenprint Integrated Services. Copyrights reserved</big></p>
+    <div class="footer1">
```

Comment:
Why do you need 2 footer wrappers @ToluEnny24

Reply:
the first wrapper is for the whole footer while the second one is for the section without the paragraph. I had issues for the mobile and tab views without he second wrapper.

Reply:
Then give it a better class name so it doesn't look confusing